packages/universal-router-sdk/src/SwapRouterPancake.ts

```diff
@@ -6,10 +6,10 @@ import { encodeFunctionData, toHex, Hex } from 'viem'
 import { PancakeSwapTrade } from './entities/protocols/pancakeswap'
 import { encodePermit } from './utils/inputTokens'
 import { RoutePlanner } from './utils/routerCommands'
-import { PancakeSwapOptions, SwapRouterConfig } from './utils/types'
+import { PancakeSwapOptions, SwapRouterConfig } from './entities/types'
 import { UniversalRouterABI } from './abis/UniversalRouter'
 
-export abstract class PancakeUniversalSwapRouter {
+export abstract class PancakeswapUniversalRouter {
```

Comment:
Open to discussion here. Which one do you guys think is better?
```suggestion
export abstract class PancakeSwapUniversalRouter {
```

Reply:
upvote

Reply:
i like PancakeSwapUniversalRouter

Reply:
ok let's switch to `PancakeSwapUniversalRouter` then

Reply:
perfect sounds good